src/Endpoints/index.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/**
+ * Loader for REST API endpoints served by this site
+ *
+ * @package Parsely
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+require_once __DIR__ . '/class-related-api-proxy.php';
```

Comment:
Interesting approach for adding a loader pattern. I like it, but I'd rather do this for all files in the project rather than just this one. 

Given that's only one file for now, could we add it as usual (in `wp-parsely.php`) and then add this structure in a further PR?